.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
```

Comment:
Is `g++-5` still used, or does using `scan-build` require us to build with clang?